client/src/utils/filtering.ts

```diff
@@ -7,10 +7,14 @@
  * for values that contain spaces.
  * Each query key has a default suffix defined e.g. name:foo will be converted to name-eq:foo.
  * Comparison aliases are allowed converting e.g. '>' to '-gt' and '<' to '-lt'.
+ *
+ * TODO: Reduce usage of `filterText` as a param in several functions
  */
 
-type Converter<T> = (value: T) => T;
-type Handler<T> = (v: T, q: T) => boolean;
+import { omit } from "lodash";
+
+type Converter<FilterValue> = (value: FilterValue) => FilterValue;
```

Comment:
This is minor, but is there a strong reason to rename the generic type `T` to `FilterValue`?

Usually `T` is generally more "standard" notation when using it as a generic type, especially if there is only one generic involved. Using `FilterValue` could be confused with an actual class or interface so if you really want to use a different name I would suggest `FilterValueType` which arguably makes more clear this is a type, otherwise I think just keeping `T` will be fine and result in fewer changes.

Reply:
I can change it back to `T`, I initially thought I could force `FilterValue = string | boolean` but then realized that would reduce reusability. I think it makes sense to revert to T